### DISCUSSION TITLE
[module] Where are the templates and plugins created? (`addPluginTemplate`, `addTemplate`)

### DISCUSSION THREAD

**Discussion by ThornWalli** [nuxt/nuxt#18621] [Questions] [answered]
Hello,

wanted to ask if physical files are still created when using `addPluginTemplate` and `addTemplate`?

In Nuxt@2 the generated plugins and templates were placed in the `.nuxt` folder. 
Now they are missing, have the suspect that here is worked with virtual modules?

**Comment by Baroshem** [comment, marked as answer]
Hey, 

You could check the Tailwind Module for Nuxt as it add physical Tailwind config to your .nuxt

**Reply by ThornWalli** [reply]
Thanks @Baroshem  for the inside 😉

Solution is: 


```js
addPluginTemplate({ ..., write: true })
```
